testing-with-jest.test.js

```diff
@@ -6,22 +6,84 @@
 // $ yarn jest
 
 
-// --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
-
-
-
-
-
-// --------------------1b) See the test fail. THEN write the code to make the test pass.
-
-
-
-
-
-// --------------------2a) Create a test for a function called triangleArea that takes the base and the height of a triange and returns the area. The area of a triange is the base times the height divided by two.
+// -----COMPLETE------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
+// almost didnt add yarn to jest before running haha
+// write out basic form of the tests
+// // create sentence saying what I want "two numbers total"
+// // needs and argument and a return
+describe("addThemUp",() =>{
+  test("takes two numbers and returns the sum",()=>{
+    expect(addThemUp(2 + 3)).toEqual(5)
+  })
+})
+// needs a function to operate, DO NOT NEED console.log();
+// just yarn jest in terminal
+const addThemUp = () =>{
+  return "sum"
+}
+//
+// // --------------------1b) See the test fail. THEN write the code to make the test pass.
+
+describe("addThemUp",() =>{
+  test("takes two numbers and returns the sum",()=>{
+    expect(addThemUp(2 + 3)).toEqual("sum")
+  })
+})
+
+const addThemUp = () =>{
+  return "sum"
+// }
+//had .toEqual(5), remember the return in the test, must be the same as the return in the function!!!
+// EX: .toEqual("sum") ------PASS
+
+
+// // --------------------2a) Create a test for a function called triangleArea that takes the base and the height of a triange and returns the area. The area of a triange is the base times the height divided by two.
+//
+// // need two arguments
+// // need three expects??
+// // return is area, base * height/2
+// describe("traingleArea",() =>{
+//   test("multiply base times height then divide by two to return the area",()=>{
+//     expect(addThemUp("base")).toEqual(5)
+//     expect(addThemUp("height")).toEqual(10)
```

Comment:
So our expects need to look like this- 

```javascript
expect(addThemUp(basenumber, heightnumber).toEqual(calculatedvalue)
```

What you have right now is passing the word "base" as a string into the function call. Same thing with the height. Both the height and the base need to be passed into the expect's function call, but they wont be words, they should be numbers.